src/components/regestration/SignUp.tsx

```diff
@@ -1,96 +1,101 @@
-import React,{useState} from "react";
-import Input from '../Input';
-import Button from "../Button";
-import CenteredContainer from "../CenteredContainer";
-import GreyContainerBox from "../GreyContainerBox";
-import Link from "../Link";
-import SecondaryText from "../SecondaryText";
-import MainRegistryContent from "./MainRegistryContent";
-import Form from "../Form";
-import {Title,DisplayTitle} from "../Titles";
-import AdditionalRegistryContent from "./AdditionalRegistryContent";
-import Logo from "../Logo";
-import { api } from "../../api";
-import { ISignUpRequest } from "../../api/intefaces";
-import ToolTip from "../ToolTip";
+import React, { useState } from 'react'
+import Input from '../Input'
+import Button from '../Button'
+import CenteredContainer from '../CenteredContainer'
+import GreyContainerBox from '../GreyContainerBox'
+import Link from '../Link'
+import SecondaryText from '../SecondaryText'
+import MainRegistryContent from './MainRegistryContent'
+import Form from '../Form'
+import { Title, DisplayTitle } from '../Titles'
+import AdditionalRegistryContent from './AdditionalRegistryContent'
+import Logo from '../Logo'
+import { api } from '../../api'
+import { ISignUpRequest } from '../../api/intefaces'
+import ToolTip from '../ToolTip'
 
-function SignUp(){
-    //Referencing to SignUpFormState interface
-    const [formDate,setFormDate] = useState<ISignUpRequest>({
-      first_name:'',
-      last_name:'',
-      email:'',
-      password:''
-    })
-  
-    //Getting key/value from input to update formDate
-    const handleChange = (e: React.ChangeEvent<HTMLInputElement>) =>{
-      const {name,value} = e.target;
-      setFormDate(previousData =>({...previousData,[name]:value}));
-    }
+function SignUp() {
+  //Referencing to SignUpFormState interface
+  const [formDate, setFormDate] = useState<ISignUpRequest>({
+    first_name: '',
+    last_name: '',
+    email: '',
+    password: '',
+  })
+
+  //Getting key/value from input to update formDate
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = e.target
+    setFormDate((previousData) => ({ ...previousData, [name]: value }))
+  }
 
   // Send formDate to server
-    const handleSubmit = (e: React.ChangeEvent<HTMLFormElement>) =>{
-      e.preventDefault();
+  const handleSubmit = (e: React.ChangeEvent<HTMLFormElement>) => {
+    e.preventDefault()
 
-      api.auth.signUp(formDate)
-        .then(data=>console.log(data));
+    api.auth.signUp(formDate).then((data) => console.log(data))
```

Comment:
It is not a change request, just a comment: If you need specific code (e.g. `console.log`) specifically for development environment, but you'd like to delete it on production environment, you can use bundlers' `__DEV__` env in this way:

```javascript
if (process.env.NODE_ENV !== 'production') {
  api.auth.signUp(formDate).then((data) => console.log(data))
}
```
It will strip this `if` block for production code to make app's bundle smaller, but will leave this code as is for development bundle.

For more info, read about environment variables for CRA, webpack or vite:
1. https://create-react-app.dev/docs/adding-custom-environment-variables/
2. https://webpack.js.org/guides/environment-variables/
3. https://vitejs.dev/guide/env-and-mode

If you want to avoid any console.log across every envs, you may consider using `no-console` rule:
https://eslint.org/docs/latest/rules/no-console

Reply:
thx, will take it into consideration